src/roiextractors/multiimagingextractor.py

```diff
@@ -127,7 +127,7 @@ def get_video(
                 start_frame=relative_start, end_frame=relative_stop
             )
 
-        video_shape = (stop - start,) + self._imaging_extractors[0].get_image_size()
+        video_shape = (stop - start,) + self.get_image_size()
```

Comment:
Why do you need to change this?

Since it's an abstract method wouldn't you need to also define `get_image_size` generally for the `MultiImagingExtractor`?

I'd prefer if you can leave the base class here unchanged and do what you need to in the child classes, is that possible?

Reply:
you're right, it's not necessary to change, better to keep it as is.